independent-publisher-connectors/Schiphol Airport/Readme.md

```diff
@@ -0,0 +1,41 @@
+# Schiphol Airport
+Schiphol Airport (AMS IATA) is one of the busiest airports in the world. It is 15 km (9.3 mi) southwest of Amsterdam in the Netherlands. 
+It serves the city of Amsterdam and the rest of the Netherlands. It is one of the major hubs in Europe with flights between many destinations in Asia, Europe and North America. 
+It is home to KLM, the Dutch flag carrier airline. As of 2016, Schiphol airport is the third busiest in Europe behind London Heathrow and Paris Charles de Gaulle and ahead of Frankfurt Airport as measured by passenger numbers.
+
+## Publisher: Michel Gueli
+
+## Prerequisites
+You need a Schiphol Developer account. You can get one for free [here](https://developer.schiphol.nl/signup).
+
+## Obtaining Credentials
+In all requests both the APP ID and APP KEY have to be provided. These settings are visible in the Dashboard, Show API Keys.
+
+## Supported Operations
+
+### Retrieve Aircraft Types
+Retrieves list of aircrafttypes. [More info](https://developer.schiphol.nl/apis/flight-api/v4/flights?version=latest#!/aircraft-type-controller/retrieveAllAircraftTypesUsingGET)
```

Comment:
@Miesjh there's a typo in `aircrafttypes`. Please make sure everything is grammatically correct and are free of typographical errors. You can refer to [the documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).